<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DivisionDialog</class>
 <widget class="QDialog" name="DivisionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="divisionComboBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="divisionListWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>371</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="analyzeButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>79</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Analyze Stuff</string>
   </property>
  </widget>
  <widget class="QLabel" name="divisionLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="divisionRulesLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
